packages/data/lib/src/key_store/store.dart

```diff
@@ -1,9 +1,14 @@
 class ApiKeyStore {
   final Map<String, dynamic> _keys;
+  final bool isProd;
 
-  const ApiKeyStore(this._keys);
+  const ApiKeyStore(
+    this.isProd,
+    this._keys,
+  );
 
-  String get traktApiKey => _keys['traktApiKey'] ?? '';
+  String get traktApiKey =>
+      isProd ? _keys['traktApiKey'] ?? '' : _keys['sandBoxApiKey'] ?? '';
```

Comment:
the same, key should be the same, but its value is different in different config files